packages/synapse-react-client/src/components/CitationPopover/useCitation.tsx

```diff
@@ -0,0 +1,35 @@
+import { useQuery } from '@tanstack/react-query'
+
+export const fetchCitation = async (
+  doi: string | undefined,
+  format: string,
+) => {
+  const formattedDoi = doi ? doi.replace('https://doi.org/', '') : ''
+  try {
+    const response = await fetch(
+      `https://citation.doi.org/format?doi=${formattedDoi}&style=${format}&lang=en-US`,
+      {
+        headers: {
+          Accept: `text/x-${format}`,
+        },
+      },
+    )
+    if (!response.ok) {
+      throw new Error(
+        `Failed to fetch citation: ${response.status}: ${response.statusText}`,
+      )
+    }
+    return await response.text()
+  } catch (err) {
+    console.log(err)
+    return ''
+  }
```

Comment:
Since this is an external API that we don't control, I think we need to do more than log and silently fail by returning an empty string. `useQuery` can handle catching a thrown error. I think you could remove the `try`/`catch` and let the component handle error UI
```suggestion
  const response = await fetch(
    `https://citation.doi.org/format?doi=${formattedDoi}&style=${format}&lang=en-US`,
    {
      headers: {
        Accept: `text/x-${format}`,
      },
    },
  )
  if (!response.ok) {
    throw new Error(
      `Failed to fetch citation: ${response.status}: ${response.statusText}`,
    )
  }
  return await response.text()
```